crates/primitives/Cargo.toml

```diff
@@ -16,6 +16,7 @@ alloy-primitives = { workspace = true, features = ["rlp"] }
 
 # Alloy Types
 alloy-sol-types = { version = "0.7.1", default-features = false }
+alloy-eips = { git = "https://github.com/alloy-rs/alloy", rev = "cb95183", default-features = false }
```

Comment:
We've been pinned to `e3f2f07` in other crates that are using `alloy-rs/alloy` - mind updating?

```suggestion
alloy-eips = { git = "https://github.com/alloy-rs/alloy", rev = "e3f2f07", default-features = false }
```

Reply:
Unfortunately build fails with that rev :( 

I used the rev `cb95183` because it's the one pinned inside op-alloy-consensus.
Want me to open a PR on your fork of op-alloy instead?

Reply:
Ah I'll update that later - for now this is fine!